models/workspacemanager/mantid_workspace_provider.py

```diff
@@ -1,17 +1,22 @@
-from WorkspaceProvider import WorkspaceProvider
+from workspace_provider import WorkspaceProvider
 from mantid.simpleapi import mtd,Load,DeleteWorkspace,Load,GroupWorkspaces,RenameWorkspace,SaveNexus
 
 class MantidWorkspaceProvider(WorkspaceProvider):
+
     def getWorkspaceNames(self):
```

Comment:
Absolutely. Remember, from the `presenters` point of view, the `MantidWorkspaceProvider`, and what it runs under-the-hood (manitd.simpleapi) is implementation detail. Follow the PEP8 standard there's no reason to let the `mantid.simpleapi` dictate the casing.